src/execute/oas3/style-serializer.js

```diff
@@ -59,10 +43,14 @@ export default function stylize(config) {
 export function valueEncoder(value, escape) {
   if (Array.isArray(value) || (value !== null && typeof value === 'object')) {
     value = JSON.stringify(value);
+  } else if (typeof value === 'number' || typeof value === 'boolean') {
```

Comment:
Added a check for `boolean` type values as I've noticed they were not being converted for headers if just using Swagger Client - the values should already be stringified when getting them from Swagger UI. For header these values will be returned without any encoding, so now the result for the `buildRequest` function will be something like:
```
{
  url: 'http://localhost:8080/',
  credentials: 'same-origin',
  headers: { headerNum: '1', headerBool: 'false' },
  method: 'POST'
}
```
and before this change it was:
```
{
  url: 'http://localhost:8080/',
  credentials: 'same-origin',
  headers: { headerNum: '1', headerBool: false },
  method: 'POST'
}
```

Reply:
Yep, as discussed last week, `number` and `boolean` types along with `null` were affected. So the check is warranted.